package.json

```diff
@@ -52,7 +52,7 @@
     "remark-docusaurus-tabs": "^0.2.0",
     "styled-components": "^6.1.8",
     "tailwindcss": "^3.3.5",
-    "vercel": "^32.5.0",
```

Comment:
@JamieSinn we arn't using vercel anymore here right? I can remove it?

Reply:
Yeah you should be fine